### PR TITLE
Issue #20: Run the Latency Test Harness against the RPC Mirror

### PR DESCRIPTION
**Pull request by creindle**
[X] Created scriptable latency test
[X] Latency test results as markdown file

Addressing issue: https://github.com/vistadataproject/TestingAndReporting/issues/20

### CODE REVIEW

**Comment by mfuroyama** [comment]
Let's take the report out of the committed repo and into a wiki, please!

**Comment by mfuroyama** [comment]
Also, let's talk about your charts.

So, for some background, I'm going to drop some Science 101 knowledge here: The purpose of a line graph, typically, is to show how changes to a single particular value (x-axis) affects another, independent value (y-axis). In the specific case of your graphs, what your graph **should** be showing is how **concurrent connections** (x-axis) affects **latency** (y-axis)

However, in the graphs you have, you're varying multiple variables (data length/rate) in a way that's confusing or misleading within the context of the individual graphs. Each line on your graph should reflect a **single** set of variable changes (x-axis).

Another basic Science 101 nugget o' knowledge here: typically, you the ticks on your x-axis to represent a linear value progression. That is, the distance between each tick should progress at consistent, equal intervals. The x-axis ticks should drive the placement of the data, not the other way around. Otherwise, your graph becomes hard to read and worse, misleading.

Given that information, one way you might want to consider representing your data is with **multiple** lines on the same line graph. Each line on the graph would represent the **concurrent connections-vs-latency** relationship at **one particular data length/rate** level.

So, basically, what I'm asking you to do is to redo the graphs:
* Consistent, evenly spaced ticks on the x-axis.
* Recreate your line graphs with multiple lines
   - Each line should show the relationship between concurrent connections vs. latency **for one level of data length/rate**
   - Keep all other variables (duration, etc.) constant for **all the test runs**

Does this make sense? If it doesn't let me know and we can discuss. This comment and the ask contained within is of way higher importance than the "move this to a wiki" comment, by the way!

**Comment by creindle** [comment]
It's been sooo long since I've created a graph... Thanks for directions!

**Comment by mfuroyama** [comment]
Wait, @creindle, you merged the results from this pull request, but the new graphs you created are still confusing, and honestly, probably not acceptable as a final product. Obvious issues I see looking at the chart:

* The legend on the right is confusing. What do those lines represent. There is no labeling there
* I only see one color line and a dotted line. Because the legend is confusing, these lines are confusing as well
* There are two points at both 200 and 300 concurrent connections on both graphs. If you did what we discussed earlier, this wouldn't be happening. I'll need you to explain to me how you got these points.
* The Markdown that would have possibly explained these results seem to have been deleted. What's going on?

* Are there any other reports other than the charts?

Let's talk about this tomorrow, but this pull request is still not approved.

**Comment by creindle** [comment]
I moved the markdown file into a wiki [page](https://github.com/vistadataproject/TestingAndReporting/wiki/Latency-Harness-Test) and I am definitely working on adding more points on the graph because the lines for the other data rates don't have enough points to visually show their effect in comparison to that main line. Also! I forgot to add the legend labels to these new graphs, but those were data rates.
